dashboard.go

```diff
@@ -591,6 +591,8 @@ type LinePlusSingleStatProperties struct {
 	DecimalPlaces     DecimalPlaces    `json:"decimalPlaces"`
 	Note              string           `json:"note"`
 	ShowNoteWhenEmpty bool             `json:"showNoteWhenEmpty"`
+	XColumn           string           `json:"xColumn"`
```

Comment:
Seems like we might need to feature flag the UI parts of the this feature, so that we can ensure these API changes go out first.